### DISCUSSION TITLE
v3.0.0 Release Planning

### DISCUSSION THREAD

**Discussion by mbordash2** [onicagroup/runway#2624] [General]
Hi everyone, we are in the planning phase for the upcoming major release of `runway` and would love any and all feedback.

Here is a snapshot of some of the major items:
- Removal of Kubernetes support.
- Simplification of the StaticSite module, removing the authentication support.
- Minimum Python version support updates.
- Cfngin blueprint improvements.

What else would you like to see in `v3`?    

For more detailed tracking and progress, see the [milestone](https://github.com/rackspace/runway/milestone/8).

**Comment by nkempel** [comment]
I wouldn't mind a new feature to store tfvars files, if not using the auto_tfvars method, within a specified directory (such as an 'env' directory) so they do not have to be in the module's root directory.  If there are a ton of .tfvars files, this can make the directory long...  Currently, at least as far as I know, the --var-file= flag used in the terraform command only acts on .tfvars files within the module's path, but it would be nice to place them all in another directory within that module's directory to keep them nice and organized. 😊
Understandably, using the "terraform_write_auto_tfvars" option is a wonderful solution to multiple terraform environment variable files, but just a thought and potentially a fairly easy new feature to add.

**Comment by lucasantarella** [comment]
Would love to see some performance/caching improvements. running this in CI/CD in Github actions takes a while as it appears the modules are fully initialized every run.

Second, would love for better homogenous output. There's alot of wrappers and poor delination between terraform outputs and such with CI/CD, and using terraform Plan is almost impossible with runway

Lastly would like to see better control over the env:/ backend support of terraform. This is handy at times but makes local debugging and development difficult since you can't use terraform init very easily.
